src/@next/components/molecules/ColorSelection/ColorSelection.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import { IProps, IColor, LocalStorageKeys, IPosition } from "./types";
+import { ColorItem } from "./ColorItem";
+import "./index.scss";
+
+const getDataFromStorage = () =>
```

Comment:
There should already be a react hook you can use for this. `useLocalStorage`, see if that would work for your use case first.

Reply:
Done